content/references/proptypes.md

```diff
@@ -78,6 +80,47 @@ export const defaultConfig = {
 
 Merchant edits to color fields are stored in RGBA format.
 
+### date
+
+![Date Picker](proptype-date.png)
+
+```javascript
+date: {
+  label: 'Date',
+  type: ElementPropTypes.date
+}
+
+export const defaultConfig = {
+  date: new Date()
+}
+```
+
+The above example is the recommended method for setting the default date as it will use the date the block is added to page.
+
+### dateRange
+
+![DateRange Picker](proptype-dateRange.png)
+
+```javascript
+dateRange: {
+  label: 'Date Range',
+  type: ElementPropTypes.dateRange
+}
+
+const today = new Date();
+const nextWeek = new Date();
+nextWeek.setDate(nextWeek + 7);
+
+export const defaultConfig = {
+  dateRange: [
+    new today,    // start date
+    new nextWeek  // end date
```

Comment:
Just realized this says
```
    new today,    // start date
    new nextWeek  // end date
```
instead of 
```
    today,    // start date
    nextWeek  // end date
```

Reply:
Well that's embarrassing… 😐 Fixed.